openchain-blockchain-coinprism/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Openchain validator (single-node)",
+  "description": "This template creates single-node Openchain validator node. The endpoint is exposed over HTTP so it is only recommended for testing and development purposes. Openchain is a Blockchain technology stack suited for organizations wishing to issue and manage digital assets in a robust, secure and scalable way. See openchain.org for more information.",
+  "summary": "This template creates a test Openchain validator node.",
+  "githubUsername": "flavien",
```

Comment:
@flavien You can sign it here https://cla.azure.com/. Can you submit this PR yourself? It helps us identify the authors later using Git history in case of issues. Also the bot will validate your CLA is signed or not when you open the PR. Thanks!